WireAuthentication/Sources/WireAuthenticationUI/Views/Login/LoginViaSSO/LoginViaSSOView.swift

```diff
@@ -18,16 +18,35 @@
 
 import SwiftUI
 
-struct LoginViaSSOView: View {
+package protocol LoginViaSSOBuilder {
+
+    @MainActor
+    func loginViaSSOView(ssoCode: UUID) -> LoginViaSSOView
+
+}
+
+package struct LoginViaSSOView: View {
 
     @ObservedObject var viewModel: LoginViaSSOViewModel
 
-    var body: some View {
-        Color.red
+    package init(
+        viewModel: LoginViaSSOViewModel
+    ) {
+        self.viewModel = viewModel
+    }
+
+    package var body: some View {
+        if let ssoURL = viewModel.ssoURL {
+            BrowserView(url: ssoURL)
+        }
```

Comment:
question: what should we show if there's no sso url?

Reply:
I changed the code, ssoURL is not optional in LoginViaSSO viewModel. The error will be handled on the previous screen in the next PR